tracks/dart/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Reasonable solutions
+
+The most simple solution is to set the default value for the parameter to `"you"` and use string interpolation:
+
+```dart
+String twofer([String name = "you"]) {
+  return "One for $name, one for me.";
+}
+```
+
+As an alternative, one could use an expression-bodied method:
+
+```dart
+String twofer([String name = "you"]) => "One for $name, one for me.";
+```
+
+An alternative solution would be to use set the default value to `null` and use the 
+[null-coalescing operator][reference-null-coalescing-operator]:
+
+```dart
+String twofer([String name = null]) {
+  return "One for ${name ?? "you"}, one for me.";
+}
+```
+
+As an alternative, one could use an expression-bodied method here too:
+
+```csharp
+String twofer([String name = null]) => "One for ${name ?? "you"}, one for me.";
+```
+
+### Common suggestions
+
+- If they are using overloaded methods, suggest to use a single method with an 
+[optional parameter][reference-optional-parameter].
+- Instead of using an if-statement, suggest to use a single expression. This will force the student 
+to remove any duplication in the code.
+- Suggest using [string interpolation][reference-string-interpolation] over string concatenation 
+approach.
+- If they are using `null` as the default value, suggest to use `"you"` instead.
+
+### Talking Suggestions
+
+- Many solutions re-assign the name parameter (e.g. `if (name == null) name = "you";`). While there 
```

Comment:
This reads as a suggestion. Not a talking point.

Instead, "discuss immutability vs mutability" and expand on that.

Reply:
I good way to rephrase this to to state something like:

> One one best practice is to avoid changing the value of the function parameters.